src/pevm.rs

```diff
@@ -161,7 +161,7 @@ impl Pevm {
         );
 
         if block_size > self.execution_results.len() {
-            let additional = block_size - self.execution_results.len();
+            let additional = block_size.saturating_sub(self.execution_results.len());
```

Comment:
It would be cleaner if we bring this above the `if`, then do `if additional > 0`.